src/main/java/com/blackducksoftware/ohcount4j/FileCount.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2016 Black Duck Software, Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.blackducksoftware.ohcount4j;
+
+public class FileCount {
+    private final SourceFile source;
+    private final LanguageCount count;
```

Comment:
I don't understand this class.  This is a mapping of a single source file to all the language counts for each language in the source file. I suspect this may not do what is needed and would expect to see a single SourceFile with a set of LanguageCounts.

We suspect that adding the use cases in test scenarios will help to clarify this class's purpose.